week1_intro/crossentropy_method.ipynb

```diff
@@ -24,8 +24,8 @@
     "\n",
     "    !touch .setup_complete\n",
     "\n",
-    "# This code creates a virtual display to draw game images on.\n",
-    "# It will have no effect if your machine has a monitor.\n",
+    "# This code creates a virtual display for drawing game images on.\n",
+    "# It won't have any effect if your machine has a monitor.\n",
```

Comment:
We have this block in (almost?) every notebook. I wouldn't stress that too much, but if this block is changed in one notebook, it should probably be changed similarly in all other notebooks.

Luckily, that can be done quite easily, since this block is absolutely identical across all notebooks.